src/components/NavigationBar/NavigationBar.tsx

```diff
@@ -0,0 +1,132 @@
+import * as React from 'react';
+import AppBar from '@mui/material/AppBar';
+import Box from '@mui/material/Box';
+import Toolbar from '@mui/material/Toolbar';
+import IconButton from '@mui/material/IconButton';
+import Typography from '@mui/material/Typography';
+import Menu from '@mui/material/Menu';
+import Container from '@mui/material/Container';
+import Button from '@mui/material/Button';
+import MenuItem from '@mui/material/MenuItem';
+import Theme from 'styles/themes/Theme';
+import { ReactComponent as Paw } from 'assets/paw-solid.svg';
+import { makeStyles } from '@mui/styles';
+import imgLogo from 'assets/logo.png';
+import 'styles/fonts.scss';
+
+const pages = ['About Us', 'Contact', 'Sign Up', 'Log In'];
+
+const useStyles = makeStyles({
+  root: {
```

Comment:
you can create a NavigationBarStyle.ts file next to NavigationBar.tsx and import the styles

Reply:
fixed - move to new file